src/sonic-yang-models/yang-models/sonic-crm.yang

```diff
@@ -0,0 +1,203 @@
+module sonic-crm {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-crm";
+    prefix crm;
+
+    import sonic-types {
+        prefix stypes;
+        revision-date 2019-07-01;
+    }
+
+    description "BREAKOUT_CFG YANG Module for SONiC OS";
+
+    revision 2020-04-10 {
+        description "First Revision";
+    }
+
+    container sonic-crm {
+
+        container CRM {
+
+            description "CRM part of config_db.json";
+
+            container Config {
+
+                /* typedef specific to CRM */
+                typedef threshold {
+                    type uint16;
```

Comment:
threshold range?

Reply:
No constraints are defined right now, except it is percentage. That why I kept below to handle the percentage part.

                    must "(((current()='PERCENTAGE' or current()='percentage') and
                        ../acl_counter_high_threshold<100 and
                        ../acl_counter_low_threshold<100) or
                        (current()!='PERCENTAGE' and current()!='PERCENTAGE'))";
                    type stypes:crm_threshold_type;